automatic/itunesfusion/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$downloadDir        = Get-PackageCacheLocation
+$installer          = 'iTunesFusionSetup-2.6.exe'
+$arguments          = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = 'iTunesFusion'
+    file            = Join-Path $downloadDir $installer
```

Comment:
only use the `file` parameter when creating embedded packages.
Remove this please